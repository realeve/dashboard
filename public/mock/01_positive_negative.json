{
  "data": [
    ["收入", "周一", 320],
    ["付出", "周一", 120],
    ["收入", "周二", 302],
    ["付出", "周二", 132],
    ["收入", "周三", 341],
    ["付出", "周三", 101],
    ["收入", "周四", 374],
    ["付出", "周四", 134],
    ["收入", "周五", 390],
    ["付出", "周五", 190],
    ["收入", "周六", 450],
    ["付出", "周六", 230],
    ["收入", "周日", 420],
    ["付出", "周日", 210]
  ],
  "title": "旋风图_读自API接口",
  "header": ["星期", "类型", "交易发生值"],
  "rows": 14,
  "hash": "mockdata"
}
