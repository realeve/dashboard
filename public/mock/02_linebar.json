{
  "data": [
    ["1月", "类目1", 175],
    ["1月", "类目2", 210],
    ["1月", "类目3", 23],
    ["2月", "类目1", 125],
    ["2月", "类目2", 140],
    ["2月", "类目3", 17.5],
    ["3月", "类目1", 98],
    ["3月", "类目2", 120],
    ["3月", "类目3", 15],
    ["4月", "类目1", 120],
    ["4月", "类目2", 140],
    ["4月", "类目3", 12],
    ["5月", "类目1", 50],
    ["5月", "类目2", 60],
    ["5月", "类目3", 8],
    ["6月", "类目1", 60],
    ["6月", "类目2", 70],
    ["6月", "类目3", 7],
    ["7月", "类目1", 50],
    ["7月", "类目2", 60],
    ["7月", "类目3", 6],
    ["8月", "类目1", 50],
    ["8月", "类目2", 55],
    ["8月", "类目3", 5],
    ["9月", "类目1", 50],
    ["9月", "类目2", 55],
    ["9月", "类目3", 5],
    ["10月", "类目1", 50],
    ["10月", "类目2", 60],
    ["10月", "类目3", 5],
    ["11月", "类目1", 40],
    ["11月", "类目2", 48],
    ["11月", "类目3", 5],
    ["12月", "类目1", 25],
    ["12月", "类目2", 30],
    ["12月", "类目3", 4]
  ],
  "title": "拆线柱图_读自API接口",
  "header": ["月份", "类型", "交易发生值"],
  "rows": 36,
  "hash": "mockdata"
}
