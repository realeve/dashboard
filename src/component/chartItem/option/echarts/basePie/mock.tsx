import type { IChartMock, IApiConfig } from '@/component/chartItem/interface';

export { config } from './config';

export const mock: IChartMock = {
  data: [
    ['1月', 175],
    ['2月', 125],
    ['3月', 98],
    ['4月', 120],
    ['5月', 50],
  ],
  title: '某数据_MOCK数据',
  header: ['类型', '交易发生值'],
  rows: 10,
  hash: 'mockdata',
};

export const apiConfig: IApiConfig = {
  show: true,
  type: 'url',
  url: 'http://localhost:8000/mock/46_base_pie.json',
  interval: 5,
  cache: 2,
  config: [
    {
      key: 'x',
      title: 'x 字段',
      defaultValue: 0,
      min: 0,
    },
    {
      key: 'y',
      title: 'y 字段',
      defaultValue: 1,
      min: 0,
    },
    {
      key: 'legend',
      title: 'legend 字段',
      defaultValue: 2,
      min: 0,
    },
  ],
};

/**
 * 默认渲染引擎
 */
// export const defaultOption = {
//   renderer: 'svg',
// };
