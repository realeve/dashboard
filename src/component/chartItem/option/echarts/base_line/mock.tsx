import type { IChartMock, IApiConfig } from '@/component/chartItem/interface';

export { config } from './config';

export const mock: IChartMock = {
  data: [
    ['类目1', '1月', 175],
    ['类目2', '1月', 210],
    ['类目1', '2月', 125],
    ['类目2', '2月', 140],
    ['类目1', '3月', 98],
    ['类目2', '3月', 120],
    ['类目1', '4月', 120],
    ['类目2', '4月', 140],
    ['类目1', '5月', 50],
    ['类目2', '5月', 60],
  ],
  title: '某数据_MOCK数据',
  header: ['月份', '类型', '交易发生值'],
  rows: 10,
  hash: 'mockdata',
};

export const apiConfig: IApiConfig = {
  show: true,
  type: 'url',
  url: 'http://localhost:8000/mock/34_base_line.json',
  interval: 5,
  cache: 2,
  config: [
    {
      key: 'legend',
      title: 'legend 字段',
      defaultValue: 0,
      min: 0,
    },
    {
      key: 'x',
      title: 'x 字段',
      defaultValue: 1,
      min: 0,
    },
    {
      key: 'y',
      title: 'y 字段',
      defaultValue: 2,
      min: 0,
    },
  ],
};

/**
 * 默认渲染引擎
 */
// export const defaultOption = {
//   renderer: 'svg',
// };
