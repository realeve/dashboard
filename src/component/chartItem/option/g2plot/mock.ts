import { IChartMock } from '@/component/chartItem/interface';
export let mock: IChartMock = {
  data: [
    { year: '周一', value: 120, category: '邮件营销' },
    { year: '周一', value: 132, category: '联盟广告' },
    { year: '周一', value: 101, category: '视频广告' },
    { year: '周一', value: 134, category: '直接访问' },
    { year: '周一', value: 90, category: '搜索引擎' },
    { year: '周二', value: 220, category: '邮件营销' },
    { year: '周二', value: 182, category: '联盟广告' },
    { year: '周二', value: 191, category: '视频广告' },
    { year: '周二', value: 234, category: '直接访问' },
    { year: '周二', value: 290, category: '搜索引擎' },
    { year: '周三', value: 150, category: '邮件营销' },
    { year: '周三', value: 232, category: '联盟广告' },
    { year: '周三', value: 201, category: '视频广告' },
    { year: '周三', value: 154, category: '直接访问' },
    { year: '周三', value: 190, category: '搜索引擎' },
    { year: '周四', value: 320, category: '邮件营销' },
    { year: '周四', value: 332, category: '联盟广告' },
    { year: '周四', value: 301, category: '视频广告' },
    { year: '周四', value: 334, category: '直接访问' },
    { year: '周四', value: 390, category: '搜索引擎' },
    { year: '周五', value: 820, category: '邮件营销' },
    { year: '周五', value: 932, category: '联盟广告' },
    { year: '周五', value: 901, category: '视频广告' },
    { year: '周五', value: 934, category: '直接访问' },
    { year: '周五', value: 1290, category: '搜索引擎' },
  ],
  title: '某数据_MOCK数据',
  header: ['year', 'value', 'category'],
  rows: 10,
  hash: 'mockdata',
};
