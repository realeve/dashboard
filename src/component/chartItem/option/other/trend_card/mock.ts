import { IChartMock } from '@/component/chartItem/interface';
export const mock: IChartMock = {
  data: [
    {
      year: '1850',
      value: 0,
      category: 'Liquid fuel',
    },
    {
      year: '1850',
      value: 54,
      category: 'Solid fuel',
    },
    {
      year: '1850',
      value: 0,
      category: 'Gas fuel',
    },
    {
      year: '1850',
      value: 0,
      category: 'Cement production',
    },
    {
      year: '1850',
      value: 0,
      category: 'Gas flarinl',
    },
    {
      year: '1851',
      value: 0,
      category: 'Liquid fuel',
    },
    {
      year: '1851',
      value: 54,
      category: 'Solid fuel',
    },
    {
      year: '1851',
      value: 0,
      category: 'Gas fuel',
    },
    {
      year: '1851',
      value: 0,
      category: 'Cement production',
    },
    {
      year: '1851',
      value: 0,
      category: 'Gas flarinl',
    },
    {
      year: '1852',
      value: 0,
      category: 'Liquid fuel',
    },
    {
      year: '1852',
      value: 57,
      category: 'Solid fuel',
    },
    {
      year: '1852',
      value: 0,
      category: 'Gas fuel',
    },
    {
      year: '1852',
      value: 0,
      category: 'Cement production',
    },
    {
      year: '1852',
      value: 0,
      category: 'Gas flarinl',
    },
    {
      year: '1853',
      value: 0,
      category: 'Liquid fuel',
    },
    {
      year: '1853',
      value: 59,
      category: 'Solid fuel',
    },
    {
      year: '1853',
      value: 0,
      category: 'Gas fuel',
    },
    {
      year: '1853',
      value: 0,
      category: 'Cement production',
    },
    {
      year: '1853',
      value: 0,
      category: 'Gas flarinl',
    },
    {
      year: '1854',
      value: 0,
      category: 'Liquid fuel',
    },
    {
      year: '1854',
      value: 69,
      category: 'Solid fuel',
    },
    {
      year: '1854',
      value: 0,
      category: 'Gas fuel',
    },
    {
      year: '1854',
      value: 0,
      category: 'Cement production',
    },
    {
      year: '1854',
      value: 0,
      category: 'Gas flarinl',
    },
    {
      year: '1855',
      value: 0,
      category: 'Liquid fuel',
    },
    {
      year: '1855',
      value: 71,
      category: 'Solid fuel',
    },
    {
      year: '1855',
      value: 0,
      category: 'Gas fuel',
    },
    {
      year: '1855',
      value: 0,
      category: 'Cement production',
    },
    {
      year: '1855',
      value: 0,
      category: 'Gas flarinl',
    },
    {
      year: '1856',
      value: 0,
      category: 'Liquid fuel',
    },
    {
      year: '1856',
      value: 76,
      category: 'Solid fuel',
    },
    {
      year: '1856',
      value: 0,
      category: 'Gas fuel',
    },
    {
      year: '1856',
      value: 0,
      category: 'Cement production',
    },
    {
      year: '1856',
      value: 0,
      category: 'Gas flarinl',
    },
    {
      year: '1857',
      value: 0,
      category: 'Liquid fuel',
    },
    {
      year: '1857',
      value: 77,
      category: 'Solid fuel',
    },
    {
      year: '1857',
      value: 0,
      category: 'Gas fuel',
    },
    {
      year: '1857',
      value: 0,
      category: 'Cement production',
    },
    {
      year: '1857',
      value: 0,
      category: 'Gas flarinl',
    },
    {
      year: '1858',
      value: 0,
      category: 'Liquid fuel',
    },
    {
      year: '1858',
      value: 78,
      category: 'Solid fuel',
    },
    {
      year: '1858',
      value: 0,
      category: 'Gas fuel',
    },
    {
      year: '1858',
      value: 0,
      category: 'Cement production',
    },
    {
      year: '1858',
      value: 0,
      category: 'Gas flarinl',
    },
    {
      year: '1859',
      value: 0,
      category: 'Liquid fuel',
    },
    {
      year: '1859',
      value: 83,
      category: 'Solid fuel',
    },
    {
      year: '1859',
      value: 0,
      category: 'Gas fuel',
    },
    {
      year: '1859',
      value: 0,
      category: 'Cement production',
    },
    {
      year: '1859',
      value: 0,
      category: 'Gas flarinl',
    },
    {
      year: '1860',
      value: 0,
      category: 'Liquid fuel',
    },
    {
      year: '1860',
      value: 91,
      category: 'Solid fuel',
    },
    {
      year: '1860',
      value: 0,
      category: 'Gas fuel',
    },
    {
      year: '1860',
      value: 0,
      category: 'Cement production',
    },
    {
      year: '1860',
      value: 0,
      category: 'Gas flarinl',
    },
    {
      year: '1861',
      value: 0,
      category: 'Liquid fuel',
    },
    {
      year: '1861',
      value: 95,
      category: 'Solid fuel',
    },
    {
      year: '1861',
      value: 0,
      category: 'Gas fuel',
    },
    {
      year: '1861',
      value: 0,
      category: 'Cement production',
    },
    {
      year: '1861',
      value: 0,
      category: 'Gas flarinl',
    },
    {
      year: '1862',
      value: 0,
      category: 'Liquid fuel',
    },
    {
      year: '1862',
      value: 96,
      category: 'Solid fuel',
    },
    {
      year: '1862',
      value: 0,
      category: 'Gas fuel',
    },
    {
      year: '1862',
      value: 0,
      category: 'Cement production',
    },
    {
      year: '1862',
      value: 0,
      category: 'Gas flarinl',
    },
    {
      year: '1863',
      value: 0,
      category: 'Liquid fuel',
    },
    {
      year: '1863',
      value: 103,
      category: 'Solid fuel',
    },
    {
      year: '1863',
      value: 0,
      category: 'Gas fuel',
    },
    {
      year: '1863',
      value: 0,
      category: 'Cement production',
    },
    {
      year: '1863',
      value: 0,
      category: 'Gas flarinl',
    },
  ],
  title: '某数据_MOCK数据',
  header: ['year', 'value', 'category'],
  rows: 1000,
  hash: 'mockdata',
};
