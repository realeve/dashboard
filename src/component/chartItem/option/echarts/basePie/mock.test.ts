import * as lib from './mock';

test('数据模拟', () => {
  expect(lib.apiConfig).toMatchSnapshot();
  expect(lib.config).toMatchSnapshot();
  expect(lib.defaultOption).toMatchSnapshot();
  expect(lib.mock).toMatchSnapshot();
});
