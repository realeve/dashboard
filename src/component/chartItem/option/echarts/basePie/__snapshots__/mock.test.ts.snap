// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`数据模拟 1`] = `
Object {
  "cache": 2,
  "config": Array [
    Object {
      "defaultValue": 0,
      "key": "x",
      "min": 0,
      "title": "x 字段",
    },
    Object {
      "defaultValue": 1,
      "key": "y",
      "min": 0,
      "title": "y 字段",
    },
    Object {
      "defaultValue": 2,
      "key": "legend",
      "min": 0,
      "title": "legend 字段",
    },
  ],
  "interval": 5,
  "show": true,
  "type": "url",
  "url": "/mock/46_base_pie.json",
}
`;

exports[`数据模拟 2`] = `
Array [
  Object {
    "defaultValue": "pie",
    "key": "chartType",
    "option": Array [
      Object {
        "title": "饼图",
        "value": "pie",
      },
      Object {
        "title": "玫瑰图",
        "value": "rose",
      },
      Object {
        "title": "玫瑰面积图",
        "value": "area",
      },
    ],
    "title": "图表类型",
    "type": "radio",
  },
  Object {
    "defaultValue": "cbpc",
    "key": "theme",
    "option": Array [
      Object {
        "desc": "分类色板1",
        "theme": Object {
          "colors10": Array [
            "#5B8FF9",
            "#61DDAA",
            "#65789B",
            "#F6BD16",
            "#6F5EF9",
            "#78D3F8",
            "#9661BC",
            "#F6903D",
            "#008685",
            "#F08BB4",
          ],
          "colors20": Array [
            "#5B8FF9",
            "#CDDDFD",
            "#61DDAA",
            "#CDF3E4",
            "#65789B",
            "#CED4DE",
            "#F6BD16",
            "#FCEBB9",
            "#6F5EF9",
            "#D3CEFD",
            "#78D3F8",
            "#D3EEF9",
            "#9661BC",
            "#DECFEA",
            "#F6903D",
            "#FFE0C7",
            "#008685",
            "#BBDEDE",
            "#F08BB4",
            "#FFE0ED",
          ],
        },
        "title": <img
          src="/img/palette/palette0.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 0,
      },
      Object {
        "desc": "分类色板2",
        "theme": Object {
          "colors10": Array [
            "#FF6B3B",
            "#626681",
            "#FFC100",
            "#9FB40F",
            "#76523B",
            "#DAD5B5",
            "#0E8E89",
            "#E19348",
            "#F383A2",
            "#247FEA",
          ],
          "colors20": Array [
            "#FF6B3B",
            "#626681",
            "#FFC100",
            "#9FB40F",
            "#76523B",
            "#DAD5B5",
            "#0E8E89",
            "#E19348",
            "#F383A2",
            "#247FEA",
            "#2BCB95",
            "#B1ABF4",
            "#1D42C2",
            "#1D9ED1",
            "#D64BC0",
            "#255634",
            "#8C8C47",
            "#8CDAE5",
            "#8E283B",
            "#791DC9",
          ],
        },
        "title": <img
          src="/img/palette/palette1.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 1,
      },
      Object {
        "desc": "分类色板3",
        "theme": Object {
          "colors10": Array [
            "#025DF4",
            "#DB6BCF",
            "#2498D1",
            "#BBBDE6",
            "#4045B2",
            "#21A97A",
            "#FF745A",
            "#007E99",
            "#FFA8A8",
            "#2391FF",
          ],
          "colors20": Array [
            "#025DF4",
            "#DB6BCF",
            "#2498D1",
            "#BBBDE6",
            "#4045B2",
            "#21A97A",
            "#FF745A",
            "#007E99",
            "#FFA8A8",
            "#2391FF",
            "#FFC328",
            "#A0DC2C",
            "#946DFF",
            "#626681",
            "#EB4185",
            "#CD8150",
            "#36BCCB",
            "#327039",
            "#803488",
            "#83BC99",
          ],
        },
        "title": <img
          src="/img/palette/palette2.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 2,
      },
      Object {
        "desc": "分类色板4",
        "theme": Object {
          "colors10": Array [
            "#FF4500",
            "#1AAF8B",
            "#406C85",
            "#F6BD16",
            "#B40F0F",
            "#2FB8FC",
            "#4435FF",
            "#FF5CA2",
            "#BBE800",
            "#FE8A26",
          ],
          "colors20": Array [
            "#FF4500",
            "#1AAF8B",
            "#406C85",
            "#F6BD16",
            "#B40F0F",
            "#2FB8FC",
            "#4435FF",
            "#FF5CA2",
            "#BBE800",
            "#FE8A26",
            "#946DFF",
            "#6C3E00",
            "#6193FF",
            "#FF988E",
            "#36BCCB",
            "#004988",
            "#FFCF9D",
            "#CCDC8A",
            "#8D00A1",
            "#1CC25E",
          ],
        },
        "title": <img
          src="/img/palette/palette3.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 3,
      },
      Object {
        "desc": "顺序色板1",
        "theme": Object {
          "colors10": Array [
            "#B8E1FF",
            "#9AC5FF",
            "#7DAAFF",
            "#5B8FF9",
            "#3D76DD",
            "#085EC0",
            "#0047A5",
            "#00318A",
            "#001D70",
          ],
          "colors20": Array [
            "#B8E1FF",
            "#9AC5FF",
            "#7DAAFF",
            "#5B8FF9",
            "#3D76DD",
            "#085EC0",
            "#0047A5",
            "#00318A",
            "#001D70",
          ],
        },
        "title": <img
          src="/img/palette/palette4.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 4,
      },
      Object {
        "desc": "顺序色板2",
        "theme": Object {
          "colors10": Array [
            "#9DF5CA",
            "#61DDAA",
            "#42C090",
            "#19A576",
            "#008A5D",
            "#006F45",
            "#00562F",
            "#003E19",
            "#002800",
          ],
          "colors20": Array [
            "#9DF5CA",
            "#61DDAA",
            "#42C090",
            "#19A576",
            "#008A5D",
            "#006F45",
            "#00562F",
            "#003E19",
            "#002800",
          ],
        },
        "title": <img
          src="/img/palette/palette5.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 5,
      },
      Object {
        "desc": "发散色板1",
        "theme": Object {
          "colors10": Array [
            "#215B77",
            "#1B9CD0",
            "#61C9FF",
            "#ABDFFF",
            "#EFF3DE",
            "#FFDE94",
            "#FFC741",
            "#D09C10",
            "#795B16",
          ],
          "colors20": Array [
            "#215B77",
            "#227BA2",
            "#1B9CD0",
            "#22BAED",
            "#61C9FF",
            "#8AD4FF",
            "#ABDFFF",
            "#C9E9FF",
            "#EFF3DE",
            "#FFE9B8",
            "#FFDE94",
            "#FFD470",
            "#FFC741",
            "#EDB40A",
            "#D09C10",
            "#A37B16",
            "#795B16",
          ],
        },
        "title": <img
          src="/img/palette/palette6.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 6,
      },
      Object {
        "desc": "发散色板2",
        "theme": Object {
          "colors10": Array [
            "#014c63",
            "#168575",
            "#0bc286",
            "#96dcb0",
            "#F2EAEA",
            "#FFA884",
            "#FF6836",
            "#D13808",
            "#7A270E",
          ],
          "colors20": Array [
            "#014c63",
            "#10686c",
            "#168575",
            "#16a37e",
            "#0bc286",
            "#65cf9b",
            "#96dcb0",
            "#c1e8c5",
            "#F2EAEA",
            "#FFC5AC",
            "#FFA884",
            "#FF895D",
            "#FF6836",
            "#F3470D",
            "#D13808",
            "#A4300C",
            "#7A270E",
          ],
        },
        "title": <img
          src="/img/palette/palette7.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 7,
      },
      Object {
        "desc": "顺序色板3",
        "theme": Object {
          "colors10": Array [
            "#d0e3ff",
            "#b4c7ec",
            "#99acd0",
            "#7f91b4",
            "#657899",
            "#4c5f7f",
            "#344766",
            "#1c314e",
          ],
          "colors20": Array [
            "#d0e3ff",
            "#b4c7ec",
            "#99acd0",
            "#7f91b4",
            "#657899",
            "#4c5f7f",
            "#344766",
            "#1c314e",
          ],
        },
        "title": <img
          src="/img/palette/palette8.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 8,
      },
      Object {
        "desc": "顺序色板4",
        "theme": Object {
          "colors10": Array [
            "#ffe244",
            "#f6bd16",
            "#d7a300",
            "#b98800",
            "#9c6f00",
            "#7f5700",
            "#633f00",
            "#482900",
          ],
          "colors20": Array [
            "#ffe244",
            "#f6bd16",
            "#d7a300",
            "#b98800",
            "#9c6f00",
            "#7f5700",
            "#633f00",
            "#482900",
          ],
        },
        "title": <img
          src="/img/palette/palette9.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 9,
      },
      Object {
        "desc": "顺序色板5",
        "theme": Object {
          "colors10": Array [
            "#8bf4f2",
            "#6ed8d6",
            "#50bcba",
            "#2fa19f",
            "#008685",
            "#006c6c",
            "#005354",
            "#003c3d",
          ],
          "colors20": Array [
            "#8bf4f2",
            "#6ed8d6",
            "#50bcba",
            "#2fa19f",
            "#008685",
            "#006c6c",
            "#005354",
            "#003c3d",
          ],
        },
        "title": <img
          src="/img/palette/palette10.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 10,
      },
      Object {
        "desc": "顺序色板6",
        "theme": Object {
          "colors10": Array [
            "#ffbb97",
            "#ff9f7c",
            "#ff8363",
            "#e8684b",
            "#c94d33",
            "#aa311c",
            "#8c1004",
            "#6f0000",
          ],
          "colors20": Array [
            "#ffbb97",
            "#ff9f7c",
            "#ff8363",
            "#e8684b",
            "#c94d33",
            "#aa311c",
            "#8c1004",
            "#6f0000",
          ],
        },
        "title": <img
          src="/img/palette/palette11.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 11,
      },
      Object {
        "desc": "顺序色板7",
        "theme": Object {
          "colors10": Array [
            "#fdd5ff",
            "#ddb6ff",
            "#be99f7",
            "#a27fda",
            "#8765be",
            "#6c4da3",
            "#523588",
            "#391e6f",
          ],
          "colors20": Array [
            "#fdd5ff",
            "#ddb6ff",
            "#be99f7",
            "#a27fda",
            "#8765be",
            "#6c4da3",
            "#523588",
            "#391e6f",
          ],
        },
        "title": <img
          src="/img/palette/palette12.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 12,
      },
      Object {
        "desc": "顺序色板8",
        "theme": Object {
          "colors10": Array [
            "#ffcffa",
            "#ffabd5",
            "#ef8bb4",
            "#d27099",
            "#b55680",
            "#993c67",
            "#7d214f",
            "#620038",
          ],
          "colors20": Array [
            "#ffcffa",
            "#ffabd5",
            "#ef8bb4",
            "#d27099",
            "#b55680",
            "#993c67",
            "#7d214f",
            "#620038",
          ],
        },
        "title": <img
          src="/img/palette/palette13.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 13,
      },
      Object {
        "desc": "顺序色板9",
        "theme": Object {
          "colors10": Array [
            "#fdedbe",
            "#ffdf80",
            "#ffcb33",
            "#ffb200",
            "#ff8c00",
            "#ff6500",
            "#e6450f",
            "#b22c00",
            "#661900",
          ],
          "colors20": Array [
            "#fdedbe",
            "#ffdf80",
            "#ffcb33",
            "#ffb200",
            "#ff8c00",
            "#ff6500",
            "#e6450f",
            "#b22c00",
            "#661900",
          ],
        },
        "title": <img
          src="/img/palette/palette14.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 14,
      },
      Object {
        "desc": "顺序色板10",
        "theme": Object {
          "colors10": Array [
            "#ffebb0",
            "#ffdf80",
            "#faca3e",
            "#e6b80b",
            "#b5ac23",
            "#6a9a48",
            "#20876b",
            "#06746b",
            "#044e48",
          ],
          "colors20": Array [
            "#ffebb0",
            "#ffdf80",
            "#faca3e",
            "#e6b80b",
            "#b5ac23",
            "#6a9a48",
            "#20876b",
            "#06746b",
            "#044e48",
          ],
        },
        "title": <img
          src="/img/palette/palette15.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 15,
      },
      Object {
        "desc": "顺序色板11",
        "theme": Object {
          "colors10": Array [
            "#d2edc8",
            "#a9dacc",
            "#75c6d1",
            "#42b3d5",
            "#3993c2",
            "#3073ae",
            "#27539b",
            "#1e3388",
            "#171e6d",
          ],
          "colors20": Array [
            "#d2edc8",
            "#a9dacc",
            "#75c6d1",
            "#42b3d5",
            "#3993c2",
            "#3073ae",
            "#27539b",
            "#1e3388",
            "#171e6d",
          ],
        },
        "title": <img
          src="/img/palette/palette16.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 16,
      },
      Object {
        "desc": "顺序色板12",
        "theme": Object {
          "colors10": Array [
            "#facdaa",
            "#f4a49e",
            "#ee7b91",
            "#e85285",
            "#be408c",
            "#942d93",
            "#6a1b9a",
            "#56167d",
            "#42105f",
          ],
          "colors20": Array [
            "#facdaa",
            "#f4a49e",
            "#ee7b91",
            "#e85285",
            "#be408c",
            "#942d93",
            "#6a1b9a",
            "#56167d",
            "#42105f",
          ],
        },
        "title": <img
          src="/img/palette/palette17.png.webp"
          style={
            Object {
              "height": 30,
            }
          }
        />,
        "value": 17,
      },
      Object {
        "colors10": Array [
          "#5B8FF9",
          "#5AD8A6",
          "#5D7092",
          "#F6BD16",
          "#E86452",
          "#6DC8EC",
          "#945FB9",
          "#FF9845",
          "#1E9493",
          "#FF99C3",
        ],
        "colors20": Array [
          "#5B8FF9",
          "#CDDDFD",
          "#5AD8A6",
          "#CDF3E4",
          "#5D7092",
          "#CED4DE",
          "#F6BD16",
          "#FCEBB9",
          "#E86452",
          "#F8D0CB",
          "#6DC8EC",
          "#D3EEF9",
          "#945FB9",
          "#DECFEA",
          "#FF9845",
          "#FFE0C7",
          "#1E9493",
          "#BBDEDE",
          "#FF99C3",
          "#FFE0ED",
        ],
        "desc": "cbpc",
        "theme": "cbpc",
        "title": "默认",
        "value": 18,
      },
      Object {
        "desc": "ECharts5",
        "theme": Object {
          "colors10": Array [
            "#5470c6",
            "#91cc75",
            "#fac858",
            "#ee6666",
            "#73c0de",
            "#3ba272",
            "#fc8452",
            "#9a60b4",
            "#ea7ccc",
          ],
          "colors20": Array [
            "#5470c6",
            "#91cc75",
            "#fac858",
            "#ee6666",
            "#73c0de",
            "#3ba272",
            "#fc8452",
            "#9a60b4",
            "#ea7ccc",
          ],
        },
        "title": "ECharts5",
        "value": 19,
      },
    ],
    "title": "主题色",
    "type": "antselect",
  },
  Object {
    "title": "图例配置",
    "type": "divider",
  },
  Object {
    "defaultValue": true,
    "key": "legendShow",
    "title": "显示图例",
    "type": "switch",
  },
  Object {
    "defaultValue": "center",
    "key": "legendAlign",
    "option": Array [
      Object {
        "title": <AlignLeftIcon
          style={
            Object {
              "color": "#fff",
            }
          }
        />,
        "value": "left",
      },
      Object {
        "title": <AlignCenterIcon
          style={
            Object {
              "color": "#fff",
            }
          }
        />,
        "value": "center",
      },
      Object {
        "title": <AlignRightIcon
          style={
            Object {
              "color": "#fff",
            }
          }
        />,
        "value": "right",
      },
    ],
    "title": "对齐",
    "type": "radio",
  },
  Object {
    "defaultValue": "top",
    "key": "legendPosition",
    "option": Array [
      Object {
        "title": <TopIcon
          style={
            Object {
              "color": "#fff",
            }
          }
          title="上方"
        />,
        "value": "top",
      },
      Object {
        "title": <RightIcon
          style={
            Object {
              "color": "#fff",
            }
          }
          title="右方"
        />,
        "value": "right",
      },
      Object {
        "title": <BottomIcon
          style={
            Object {
              "color": "#fff",
            }
          }
          title="下方"
        />,
        "value": "bottom",
      },
      Object {
        "title": <LeftIcon
          style={
            Object {
              "color": "#fff",
            }
          }
          title="左方"
        />,
        "value": "left",
      },
    ],
    "title": "位置",
    "type": "radio",
  },
  Object {
    "defaultValue": "horizontal",
    "key": "legendOrient",
    "option": Array [
      Object {
        "title": "水平",
        "value": "horizontal",
      },
      Object {
        "title": "垂直",
        "value": "vertical",
      },
    ],
    "title": "排列方式",
    "type": "radio",
  },
  Object {
    "defaultValue": 20,
    "key": "innerRadius",
    "max": 70,
    "min": 0,
    "step": 5,
    "title": "圆环大小",
    "type": "range",
  },
  Object {
    "defaultValue": "inner",
    "key": "labelPosition",
    "option": Array [
      Object {
        "title": "内部",
        "value": "inner",
      },
      Object {
        "title": "外部",
        "value": "outer",
      },
    ],
    "title": "标签位置",
    "type": "radio",
  },
  Object {
    "defaultValue": "edge",
    "key": "labelAlign",
    "option": Array [
      Object {
        "title": "默认",
        "value": "none",
      },
      Object {
        "title": "文字对齐",
        "value": "labelLine",
      },
      Object {
        "title": "两边对齐",
        "value": "edge",
      },
    ],
    "title": "标签对齐",
    "type": "radio",
  },
  Object {
    "defaultValue": 10,
    "key": "edgeDistance",
    "max": 30,
    "min": 0,
    "step": 1,
    "title": "标签边距",
    "type": "range",
  },
  Object {
    "defaultValue": 1,
    "key": "borderWidth",
    "max": 30,
    "min": 0,
    "step": 1,
    "title": "线宽",
    "type": "range",
  },
  Object {
    "defaultValue": "#070125",
    "key": "borderColor",
    "title": "边框线颜色",
    "type": "purecolor",
  },
  Object {
    "defaultValue": 0,
    "key": "borderRadiusInner",
    "max": 50,
    "min": 0,
    "step": 1,
    "title": "内圆圆角",
    "type": "range",
  },
  Object {
    "defaultValue": 0,
    "key": "borderRadiusOutter",
    "max": 50,
    "min": 0,
    "step": 1,
    "title": "外圆圆角",
    "type": "range",
  },
  Object {
    "defaultValue": 16,
    "key": "labelFontSize",
    "max": 40,
    "min": 10,
    "step": 1,
    "title": "标签字号",
    "type": "range",
  },
  Object {
    "defaultValue": 12,
    "key": "valueFontSize",
    "max": 40,
    "min": 10,
    "step": 1,
    "title": "标签字号",
    "type": "range",
  },
]
`;

exports[`数据模拟 3`] = `
Object {
  "data": Array [
    Array [
      "1月",
      175,
    ],
    Array [
      "2月",
      125,
    ],
    Array [
      "3月",
      98,
    ],
    Array [
      "4月",
      120,
    ],
    Array [
      "5月",
      50,
    ],
  ],
  "hash": "mockdata",
  "header": Array [
    "类型",
    "交易发生值",
  ],
  "rows": 10,
  "title": "某数据_MOCK数据",
}
`;
